src/Components/CreateImageWizard/formComponents/TargetEnvironment.js

```diff
@@ -32,129 +37,203 @@ const TargetEnvironment = ({ label, isRequired, ...props }) => {
-            aria-label="Bare metal installer checkbox"
-            id="checkbox-image-installer"
-            name="Bare metal installer"
-            data-testid="checkbox-image-installer"
+          <Tile
+            className="tile pf-u-mr-sm"
+            data-testid="upload-azure"
+            title="Microsoft Azure"
+            icon={
+              <img
+                className="provider-icon"
+                src={
+                  '/apps/frontend-assets/partners-icons/microsoft-azure-short.svg'
+                }
+              />
+            }
+            onClick={() => handleSetEnvironment('azure', !environment.azure)}
+            onKeyDown={(e) => handleKeyDown(e, 'azure', !environment.azure)}
+            onMouseEnter={() => prefetchSources('azure')}
+            isSelected={environment.azure}
+            isStacked
+            isDisplayLarge
           />
-        </FormGroup>
+        </div>
       </FormGroup>
-    </>
+      <FormGroup
+        label={<Text component={TextVariants.small}>Private cloud</Text>}
+        data-testid="target-private"
+      >
+        <Checkbox
+          label="VMWare vSphere"
+          isChecked={environment.vsphere || environment['vsphere-ova']}
+          onChange={(checked) => {
+            handleSetEnvironment('vsphere-ova', checked);
+            handleSetEnvironment('vsphere', false);
+          }}
+          aria-label="VMWare checkbox"
+          id="checkbox-vmware"
+          name="VMWare"
+          data-testid="checkbox-vmware"
+        />
+      </FormGroup>
+      <FormGroup
+        className="pf-u-mt-sm pf-u-mb-sm pf-u-ml-xl"
+        data-testid="target-private-vsphere-radio"
+      >
+        <Radio
+          name="vsphere-radio"
+          aria-label="VMWare vSphere radio button OVA"
+          id="vsphere-radio-ova"
+          label={
+            <>
+              Open virtualization format (.ova)
+              <Popover
+                maxWidth="30rem"
+                position="right"
+                bodyContent={
+                  <TextContent>
+                    <Text>
+                      An OVA file is a virtual appliance used by virtualization
+                      platforms such as VMWare vSphere. It is a package that
+                      contains files used to describe a virtual machine, which
+                      includes an VMDK image, OVF descriptor file and a manifest
+                      file.
+                    </Text>
+                  </TextContent>
+                }
+              >
+                <HelpIcon className="pf-u-ml-sm" />
+              </Popover>
+            </>
+          }
+          onChange={(checked) => {
+            handleSetEnvironment('vsphere-ova', checked);
+            handleSetEnvironment('vsphere', !checked);
+          }}
+          isChecked={environment['vsphere-ova']}
+          isDisabled={!(environment.vsphere || environment['vsphere-ova'])}
+        />
+        <Radio
```

Comment:
Just a thought - could there be a scenario when the user needs to create both types of VMWare images and we would need checkboxes instead of a radio button?

Reply:
I thought radios were more apprioriate, because the ova is just a tar file which contains the vmdk inside of it. So you can just extract the vmdk with tar -xvf composer-api.ova.

Reply:
Aah, got it! Radios make perfect sense then. :+1: